lib/scarpe/flow.rb

```diff
@@ -1,7 +1,24 @@
 class Scarpe
-    class Flow < Scarpe::Widget
-      def element
-        "<div style='display: flex; flex-direction: row' id='#{html_id}'></div>"
+  class Flow < Scarpe::Widget
+    include Scarpe::Background
+
+    def initialize(&block)
+      instance_eval(&block)
```

Comment:
Is flow special in that it needs this initializer whereas stack does not?

Reply:
@Schwad Stack [does the same thing](https://github.com/Schwad/scarpe/blob/d9e587b73eec3dfc7cd8a5bd23e94160f14942a2/lib/scarpe/stack.rb#L6). Or am I misunderstanding something?